### ISSUE TITLE
training data

### ISSUE THREAD

**Issue by ssetty** [reachsumit/deep-unet-for-satellite-image-segmentation#4]
I want to know how training data with label is given to neural network. I guess here do not have any label,
How is X and Y data given? I see some 'mask' here. Please explain.

Thanks for article.

**Comment by reachsumit** [comment]
Hope you have figured this out by now. I use masks as the response (labels).
